css/style.css

```diff
@@ -0,0 +1,189 @@
+body {
+  padding: 0 50px;
```

Comment:
Removing the padding on the body will help you achieve a full-screen image background